web/thesauruses/haskell/control_structures.json

```diff
@@ -6,92 +6,49 @@
   },
   "categories": {
     "Conditional Statements (ifs)": [
-      "if_conditional",
       "if_else_conditional",
-      "if_elseif_conditional",
       "if_elseif_else_conditional",
       "ternary_conditional"
     ],
     "Loops": [
       "while_loop",
-      "do_while_loop",
       "until_loop",
-      "do_until_loop",
-      "for_loop",
       "foreach_loop"
     ],
     "Iterations": [
-      "each_iteration",
       "map_iteration",
       "filter_iteration",
-      "fold_iteration",
-      "recursion_iteration"
+      "fold_iteration"
     ]
   },
   "control_structures": {
-    "if_conditional": {
-      "name": "If conditional",
-      "code": ""
-    },
```

Comment:
I'm removing a bunch of these code blocks cause they result in noise on the reference sheets and absent keys are now handled by the back end directly.